pkg/sql/sem/builtins/builtins.go

```diff
@@ -3064,6 +3064,28 @@ may increase either contention or retry errors, or both.`,
 		},
 	),
 
+	"getdatabaseencoding": makeBuiltin(
+		tree.FunctionProperties{Category: categorySystemInfo},
+		tree.Overload{
+			Types:      tree.ArgTypes{},
+			ReturnType: tree.FixedReturnType(types.String),
+			Fn: func(ctx *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
+				// computing encoding name bypassing encoding_id which is
```

Comment:
Can you change this comment to

"Compute encoding name by calling pg_encoding_to_char on the encoding from pg_database."

All comments should be complete sentences!